src/uct/ugni/rdma/ugni_rdma_iface.c

```diff
@@ -12,7 +12,7 @@
 
 static ucs_config_field_t uct_ugni_rdma_iface_config_table[] = {
     /* This tuning controls the allocation priorities for bouncing buffers */
-    { "", "MAX_SHORT=2048;MAX_BCOPY=2048;ALLOC=huge,mmap,heap", NULL,
+    { "", "ALLOC=huge,mmap,heap", NULL,
```

Comment:
why did you remove `MAX_BCOPY` also?

Reply:
UGNI doesn't use `MAX_SHORT` and `MAX_BCOPY` config attributes

Reply:
ok